model/inline/src/main/scala/aqua/model/inline/ArrowInliner.scala

```diff
@@ -297,9 +313,9 @@ object ArrowInliner extends Logging {
     arrowsAcc: Map[String, FuncArrow] = Map.empty
```

Comment:
This function is not tail-recursive. Lets simplify it and remove accumulators

Reply:
I believe they never store anything. Each branch either pass them untouched or return result non-recursively